stencil-library/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### 3.2.5
```

Comment:
Since this is a Storybook change only, it should not be a new version (a dev would not get anything by upgrading from 3.1.5 to 3.2.5 in their application)